nin/backend/compress.js

```diff
@@ -92,8 +94,14 @@ function compress(projectPath, payload, callback) {
           'demo=bootstrap({layers:layers, camerapaths:camerapaths, onprogress: ONPROGRESS, oncomplete: ONCOMPLETE});' +
         '}' +
       '}' +
-    '}</script><canvas class=hide height='+ height + ' width=' + width + '></canvas><img src=# onload=z()><!--';
+    '}</script>' +
+    '<canvas class=hide height='+ height + ' width=' + width + '></canvas><img src=# onload=z()><!--';
   var htMlChunk = chunk('htMl', new Buffer(html));
+  var metadataChunk = chunk('iTXt', new Buffer(
```

Comment:
`iTXt` chunks should probably follow the format in the spec, see http://www.libpng.org/pub/png/spec/1.2/PNG-Chunks.html (4.2.3.3. iTXt International textual data). You can use positiveNumberToBytes to get correct widths for the different fields.